### DISCUSSION TITLE
What is the purpose of the Symlink step in the tutorial for creating a plugin?

### DISCUSSION THREAD

**Discussion by tristanfazio** [elgatosf/elgato#93] [Help] [answered]
Don't quite understand this step. What purpose does it serve?
So far to me it seems like the dev loop is to code, build a release plugin, then add it to the stream deck. So what did this symlink step do?

**Comment by ChekTek** [comment, marked as answer]
Hi @tristanfazio, 

Using a symlink allows developers to keep their projects stored wherever they like on their file system and still reload the plugin. You don't have to build a release plugin to see your code changes. If you navigate to your plugin on the debug page, you can reload the plugin on the fly. <http://localhost:23654> This works great for legacy javascript plugins running in the Qt webengine, but we now also offer an SDK beta, which allows plugins to run in a Node.js environment.

If you are interested in the beta, you can install our CLI using NPM: https://www.npmjs.com/package/@elgato/cli. This should provide a much nicer developer experience. Let me know if you have any questions! `requires Stream Deck 6.4+`

**Comment by tristanfazio** [comment]
Hi @ChekTek!

Thanks for the reply.

So it seems I have misunderstood the docs then. 

You sym link development repo folder, to the  C:\Users\%USERNAME%\AppData\Roaming\Elgato\StreamDeck\Plugins\com.example.my-plugin.sdPlugin, so that the Streamdeck can pick it up.

Thank you for clarifying!
